apps/python/scart.py

```diff
@@ -759,6 +782,16 @@ def readStreamTimeList(listFile):
   --nslc arg        Use a stream list file for filtering the data by the given
                     streams. For dump mode only! One line per stream.
                     Format: NET.STA.LOC.CHA
+  --rename arg      Rename stream data accordingly to the provided rule(s).
+                    A rule is "[match-stream:]rename-stream" and match-stream
+                    is optional. match-stream and rename-stream are in the 
+                    "NET.STA.LOC.CHA" format. match-stream supports special
+                    charactes "?" "*" "|" "(" ")". rename-stream supports the
```

Comment:
supports the regular expressions

Reply:
I would not say regular expression because that syntax is not a true regular expression. Indeed the code translates the special characters provided by the user into a valid regular expression.

Reply:
fine with me